core/config.go

```diff
@@ -53,6 +54,21 @@ func (b BodyConfig) CreatePrompt(stdinReader io.ReadCloser) Prompt {
 	return p
 }
 
+type NewLinesConfig struct {
+	BeforeVersion    int64 `yaml:"beforeVersion,omitempty"`
+	AfterVersion     int64 `yaml:"afterVersion,omitempty"`
+	BeforeComponent  int64 `default:"1" yaml:"beforeComponent,omitempty"`
+	AfterComponent   int64 `yaml:"afterComponent,omitempty"`
+	BeforeHeaderFile int64 `default:"1" yaml:"beforeHeaderFile,omitempty"`
+	AfterHeaderFile  int64 `yaml:"afterHeaderFile,omitempty"`
+	BeforeFooterFile int64 `default:"1" yaml:"beforeFooterFile,omitempty"`
+	AfterFooterFile  int64 `yaml:"afterFooterFile,omitempty"`
+	BeforeKindHeader int64 `default:"1" yaml:"beforeKindHeader,omitempty"`
+	AfterKindHeader  int64 `yaml:"afterKindHeader,omitempty"`
+	BeforeChanges    int64 `default:"1" yaml:"beforeChanges,omitempty"`
+	AfterChanges     int64 `yaml:"afterChanges,omitempty"`
```

Comment:
I think the singular form of `Change` is better to match all the other options.

Reply:
I am sorry, English is not my native language, can you simplify what you said? 
I didn't understand what you meant

Reply:
Yea, swap to `BeforeChange` and `AfterChange` where change represents one single change and changes would be more than one.